server/tests/src/test/java/org/infinispan/server/security/authorization/AbstractAuthorization.java

```diff
@@ -263,29 +264,50 @@ public void testRestNonAdminsMustNotModifyLoggers() {
    }
 
    @Test
-   public void testRestNonAdminsMustNotObtainReport() {
+   public void testRestAdminsShoudleBeAbleToAdminServer() {
+      RestClientConfigurationBuilder adminConfig = restBuilders.get("admin");
+      assertEquals(204, sync(getServerTest().rest().withClientConfiguration(adminConfig).get().server().connectorStop("endpoint-alternate-1")).getStatus());
```

Comment:
`report()` test is missing.

Reply:
That's because for report to work, we need to have the `report.sh` script available and that is not true currently for these integration tests. I'll look into it as part of my authz work